src/test/typescript-service-helpers.ts

```diff
@@ -75,18 +81,18 @@ export function describeTypeScriptService(createService: TypeScriptServiceFactor
 	describe('Workspace without project files', function (this: TestContext) {
 
 		beforeEach(initializeTypeScriptService(createService, new Map([
-			['file:///a.ts', 'const abc = 1; console.log(abc);'],
-			['file:///foo/b.ts', [
+			['a.ts', 'const abc = 1; console.log(abc);'],
+			['foo/b.ts', [
```

Comment:
Please revert this, I changed this on purpose. The LS shouldn't assume files are always file:// URIs, this will just make it harder to test non-file URIs in the future. The test itself also shouldn't rely on helper functions like `path2Uri`